pkg/setup/flags.go

```diff
@@ -16,6 +16,7 @@ type Flags struct {
 	insecure     bool
 	loginFlags   *login.Flags
 	fs           afero.Fs
+	noTimeout    bool
```

Comment:
I'd rather call this `noTimeouts` (plural) to make it clear to the user that this flag unsets all of the application-level timeout values.